From 3478a7ae7a2db5e13ed6fa8456654cba2e6d60c5 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 15 Jun 2019 14:51:06 +0200
Subject: [PATCH 1/1] efi_loader: system table setup

When setting up the system table avoid superfluous void * conversions.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_boottime.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/lib/efi_loader/efi_boottime.c b/lib/efi_loader/efi_boottime.c
index b26291b919..d104cc6b31 100644
--- a/lib/efi_loader/efi_boottime.c
+++ b/lib/efi_loader/efi_boottime.c
@@ -3620,11 +3620,11 @@ struct efi_system_table __efi_runtime_data systab = {
 	},
 	.fw_vendor = firmware_vendor,
 	.fw_revision = FW_VERSION << 16 | FW_PATCHLEVEL << 8,
-	.con_in = (void *)&efi_con_in,
-	.con_out = (void *)&efi_con_out,
-	.std_err = (void *)&efi_con_out,
-	.runtime = (void *)&efi_runtime_services,
-	.boottime = (void *)&efi_boot_services,
+	.con_in = &efi_con_in,
+	.con_out = &efi_con_out,
+	.std_err = &efi_con_out,
+	.runtime = &efi_runtime_services,
+	.boottime = &efi_boot_services,
 	.nr_tables = 0,
 	.tables = NULL,
 };
-- 
2.20.1

